scripts/policygen/sources/jira/template.yml

```diff
@@ -0,0 +1,42 @@
+source:
+  host: {{ config.jira_domain }}
+  auth: basic_auth:jira
+  rules:
+    - route: "GET /rest/api/3/search"
+      actions:
+      - kind: nullify
+        json_paths:
+        - '$.issues[*].changelog.histories[*].items[?(@.field = "summary")].fromString'
```

Comment:
Double checking here: Do we need to consider sensitive information stored within custom fields?

Reply:
I'll ask Bitgo. Before we have you implement this I'm going to get their review of this general approach.